### ISSUE TITLE
Component Request: 6 Pin Potentiometer

### ISSUE THREAD

**Issue by Miserlou** [bancika/diy-layout-creator#638]
Would love to have a 6-pin (2x3 row) potentiometer!

**Comment by KENTUCKYTUKKER** [comment]
Do you mean a stereo potentiometer. I created one by myself in using twice the potmeter symbol and add a mechanical connection line. Just name them i.e. VR5a/VR5b

**Comment by bancika** [comment]
Yes, two potentiometer symbols is the "official" way :) You can stack them together and save as a building block.
